angular/projects/catalyst/package.json

```diff
@@ -12,7 +12,8 @@
   "peerDependencies": {
     "@angular/core": ">=14.0.0",
     "@angular/cdk": ">=14.0.0",
-    "rxjs": ">=6.6.0"
+    "rxjs": ">=6.6.0",
+    "@ngx-formly/core": ">=6.1.8"
```

Comment:
Any idea how to resolve this? We now force the user to install ngx-formly because otherwise there is a dependency issue. Ideally it would be an optional dependency, but I have no idea ho that works. Because you still need it to prevent compile errors

Reply:
propably a solution is to split the library smaller libraries like. catalyst/components, catalyst/forms etc